composer.json

```diff
@@ -12,12 +12,14 @@
         "php": ">=7.4",
         "php-http/discovery": "^1.14",
         "psr/simple-cache": "^1.0",
+        "guzzlehttp/guzzle": "^7.3",
```

Comment:
This shouldnt be in both require-dev and dev and additionally because you have listed the PSRs below you dont need to (and shouldnt?) depend strictly on guzzle